gguf-py/gguf/gguf_writer.py

```diff
@@ -714,6 +714,9 @@ def add_rope_scaling_orig_ctx_len(self, value: int) -> None:
 
     def add_rope_scaling_finetuned(self, value: bool) -> None:
         self.add_bool(Keys.Rope.SCALING_FINETUNED.format(arch=self.arch), value)
+        
+    def add_mamba_dt_b_c_rms(self, value: bool) -> None:
```

Comment:
```suggestion
    def add_ssm_dt_b_c_rms(self, value: bool) -> None:
```

The use(s) of that function should be updated too if renamed.

This definition could also be moved closer to the other `add_ssm_*` functions below (ideally in the same order as in `constants.py`).

Reply:
Makes sense, done !